service/resource/cloudformationv2/adapter/launch_configuration.go

```diff
@@ -81,15 +79,10 @@ func (l *launchConfigAdapter) getLaunchConfiguration(customObject v1alpha1.AWSCo
 	l.SecurityGroupID = *output.SecurityGroups[0].GroupId
 
 	// cloud config
-	resp, err := clients.IAM.GetUser(&iam.GetUserInput{})
+	accountID, err := AccountID(clients)
```

Comment:
Nitpick but the cloud config comment is a bit confusing.

Reply:
Good point, I've modified it to make clear that it is related to the logic needed to determine the small cloud config field.